content/en/blog/falcosidekick-response-engine-part-8-fluxv2.md

```diff
@@ -0,0 +1,243 @@
+export GITHUB_TOKEN=<token>
+export GITHUB_USER=<username>
+```
+
+Once we defined the necessary environment variables, we can install Flux v2. This command will create and push its manifests to the repository, then install Flux components.
+
+```shell
+flux bootstrap github \
+  --owner=$GITHUB_USER \
+  --repository=fleet-infra \
+  --branch=main \
+  --path=./clusters/my-cluster \
+  --personal
+```
+
+After we installed the Flux v2 components, the next step is creating our  [GitRepository](./alpine-gitsource.yaml) and [Kustomization](./alpine-kustomization.yaml) CRD's (Custom Resource Definitions). If you want to learn more about them, you can visit this [link](https://fluxcd.io/docs/components/).
+
+You can apply these files like the following:
+
+```shell
+kubectl apply -f alpine-gitsource.yaml -f alpine-kustomization.yaml
+```
+
+or you can use the [Flux CLI](https://fluxcd.io/docs/cmd/#installation) to do the same thing.
+
+To create `GitRepository`:
+
+```shell
+flux create source git alpine \
+    --url https://github.com/developer-guy/desired-state-repository \
+    --branch master \
+    --interval 30s
+```
+
+To create `Kustomization`:
+
+```shell
+flux create kustomization alpine \
+    --source alpine \
+    --path "./" \
+    --prune true \
+    --validation client \
+    --interval 5m
+```
+
+### Install falco-event-listener
+
+As the name suggests, this project will listen to an event in form of `CloudEvents` that forwarded from Falcosidekick which is a simple daemon for enhancing available outputs for Falco. Once, an event received from Falcosidekick, this project will update the YAML definition to scale its replicas to zero based on the informations in the event.
+
+At the end of the day, the architecture of the demo is going to be:
+
+Falco w/HTTP --> Falcosidekick w/CloudEvent --> falco-event-listener
+
+If you want to learn more about `CloudEvents`, you can follow this [link](https://cloudevents.io).
+
+Let's clone the project's repository first.
+
+```shell
+git clone https://github.com/developer-guy/falco-event-listener
+cd falco-event-listener
+```
+
+This project is basically a CLI application and uses the following [arguments](https://github.com/developer-guy/falco-event-listener/blob/master/falcoeventlistener.yaml#L12):
+
+```yaml
+args:
+  - "--owner"
+  - "developer-guy" # this is the owner of the desired state repository
+  - "--repository"
+  - "desired-state-repository" # this is the repository which we store desired state configurations
+  - "--file"
+  - "alpine.yaml" # this is the file that we are going to update
+  - "--github-token"
+  - "$(GITHUB_TOKEN)"
+```
+
+Before installing this project, we have to [create a GitHub PAT (Personal Access Token)](https://docs.github.com/en/github/authenticating-to-github/keeping-your-account-and-data-secure/creating-a-personal-access-token) to update the desired state configurations which is in GitHub repository after we detect a malicious behaviour related to our Pod.
```

Comment:
Before installing `falco-event-listener`, we have to [create a GitHub PAT (Personal Access Token)](https://docs.github.com/en/github/authenticating-to-github/keeping-your-account-and-data-secure/creating-a-personal-access-token). This token updates the desired state configurations which are in the GitHub repository if a malicious behavior related to our pod has been detected.

Reply:
thank you so much for fixing the issues, done.